Makefile

```diff
@@ -9,9 +9,9 @@ ifeq (test_one,$(firstword $(MAKECMDGOALS)))
 endif
 
 clean:
-	find -name '*.pyc' -delete
-	find -name '*~' -delete
-	find -name '__pycache__' -type d -delete
+	find . -name '*.pyc' -delete
+	find . -name '*~' -delete
+	find . -name '__pycache__' -type d -delete
```

Comment:
@antoviaque There's actually more changes that need doing to make it work properly on Mac OS X. I was just confused on how this ever worked, since I didn't think this syntax worked in Linux either.

Where's the vagrant info? I'd be happy to drop this change and just use that instead.